sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -919,6 +903,10 @@ async def update_subscription(
 
         Before calling this method, you should use `get_subscription`, `update_subscription` or `list_subscription`
         to get a `SubscriptionProperties` instance, then update the properties.
+        You could also pass keyword arguments for updating properties in the form of
+        `<property_name>=<property_value>` which will override whatever was specified in
```

Comment:
should these keywords be documeneted?

Reply:
I was trying to be vague here -- I'm not sure whether those keywords should be documented.
as once documented, then it feels like we're promoting using keyword arguments to do updates -- two ways to do the same thing.

@annatisch , do you think we'd better document those keyword arguments?

Reply:
hmm I don't feel strongly either way.... documentation can always be updated as needed.
You could add an example in the code snippet to show their use rather than call them all out explicitly.

Reply:
code snippets updated!

Reply:
code snippets updated!